### ISSUE TITLE
User can not get tickets assigned after LDAP sync

### ISSUE THREAD

**Issue by anicoa** [zammad/zammad#1224]
- Used Zammad version: 1.5.0
- Used Zammad installation source: Ubuntu packages
- Operating system: Ubuntu 16.04 LTS
- Browser + version: Safari 10.1.1.

### Expected behavior:

* After LDAP synchronisation/user import user can get ticket assigned 

### Actual behavior:

* user can get ticket assigned only after admin changes user in at least one single attribute, e.g. admin adds phone number and save changes

**Comment by thorsteneckel** [comment]
Hi @anicoa - Thanks for reporting this. I was able to reproduce it. I'll come back to you with a fix.

**Comment by jacotec** [comment]
Hi @thorsteneckel ,
is there a status update of this? Sounds like a serious issue ...

**Comment by thorsteneckel** [comment]
Hi there. This is somehow a caching issue in the frontend which sits deeper and isn't caused by the LDAP functionality itself. We are still working on other issues but it's in our list. In some cases it helped to refresh the frontend. Just as a temporary workaround.

**Comment by thorsteneckel** [comment]
Since it's not directly caused by the LDAP sync, this issue gets closed. There is another issue addressing it but I wasn't able to find it currently.
